tests/unit_tests/docmaps/v2/test_data.py

```diff
@@ -62,7 +65,8 @@
     'preprint_version': PREPRINT_VERSION_2,
     'preprint_published_at_date': date.fromisoformat('2021-02-02'),
     'tdm_path': TDM_PATH_2,
-    'evaluations': []
+    'evaluations': [],
+    'manuscript_published_date': MANUSCRIPT_PUBLISHED_DATE_2
 }
 
 MANUSCRIPT_VOR_VERSION_1: dict = {
```

Comment:
Maybe the test could use `ApiManuscriptVersionInput` and related types? It could be another way to use mypy to check whether the type is correctly defined or whether the test data accurately reflects the expected data.